x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/fail.test.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act } from 'react-dom/test-utils';
+import { setup, SetupResult } from './processor.helpers';
+
+// Default parameter values automatically added to the URI parts processor when saved
+const defaultFailParameters = {
+  ignore_failure: undefined,
+  if: undefined,
+  tag: undefined,
+  description: undefined,
+};
+
+describe('Processor: Fail', () => {
+  let onUpdate: jest.Mock;
+  let testBed: SetupResult;
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  beforeEach(async () => {
+    onUpdate = jest.fn();
+
+    await act(async () => {
+      testBed = await setup({
+        value: {
+          processors: [],
+        },
+        onFlyoutOpen: jest.fn(),
+        onUpdate,
+      });
+    });
+    testBed.component.update();
+  });
+
+  test('prevents form submission if required fields are not provided', async () => {
+    const {
+      actions: { addProcessor, saveNewProcessor, addProcessorType },
+      form,
+    } = testBed;
+
+    // Open flyout to add new processor
+    addProcessor();
+    // Click submit button without entering any fields
+    await saveNewProcessor();
+
+    // Expect form error as a processor type is required
+    expect(form.getErrorsMessages()).toEqual(['A type is required.']);
```

Comment:
I think we should come to a resolution on @cjcenizal comment (https://github.com/elastic/kibana/pull/95130#discussion_r600489090) before merging this test.

The "type" field is common to all processors so it does not make sense to have this test repeated across different processor tests. I think extracting it to a test that is non-processor specific in its focus would be best. Let me know what you think!

Reply:
I agree with @jloleysens that it's not necessary to test for type field error and since `https://github.com/elastic/kibana/blob/master/x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/processor_form.test.tsx` has already been added in [#95130](https://github.com/elastic/kibana/pull/95130), I'd suggest simply deleting line `#56 await saveNewProcessor();` and line`#58 expect(form.getErrorsMessages()).toEqual(['A type is required.']);`. 

Not directly related to this, but I noticed in the linked `processor_form.test.tsx` file that the describe block is titled `Processor: Bytes` which should probably be something general.